.github/workflows/artifacts-summary.lock.yml

```diff
@@ -139,6 +139,19 @@ jobs:
           node-version: '24'
       - name: Install GitHub Copilot CLI
         run: npm install -g @github/copilot@0.0.347
+      - name: Install awf binary
+        run: |
+          LATEST_TAG=$(gh release view --repo githubnext/gh-aw-firewall --json tagName --jq .tagName)
```

Comment:
we should hardcode the version in gh-aw so these version stay in sync, so no magic here